### ISSUE TITLE
Dosen't work on Safari

### ISSUE THREAD

**Issue by Balooforever** [baptistecdr/aria2-extensions#126]
Hi,

The extension doesn't work for me, I can save but can't send a link to the app : nw_connection_report_state_with_handler_on_nw_queue [C10] reporting state cancelled error Connection refused

I don't understand why, Firefox Extension work great in my system.

Can you help me ?

Thank :)

**Comment by baptistecdr** [comment]
Hello,

Sorry for this problem. For the moment, the extension on Safari is unmaintained and it was more a PoC (Proof of Concept) than a real release.

I will work on it after I release the v4 of the extension, which should be ready in June I hope.

In the mean time you can use the Firefox extension.

Have a nice day,
baptistecdr

**Comment by Balooforever** [comment]
Ok, thank for your work :)

**Comment by baptistecdr** [comment]
Did you try this version ? https://github.com/baptistecdr/aria2-extensions/issues/30#issuecomment-1038158464

**Comment by Balooforever** [comment]
Yes, same problem : can t send to aria.

In debug on Xcode i show « can t open file »

**Comment by myd986** [comment]
When I click "download with aria2" on Safari, it shows "failed to add like". Same setting works will on Chrome.
<img width="627" alt="Screenshot 2022-10-18 at 11 26 07" src="https://user-images.githubusercontent.com/49913958/196328659-fd129f04-035d-4e21-8525-14ad5ef3b2dc.png">
<img width="370" alt="Screenshot 2022-10-18 at 11 01 17" src="https://user-images.githubusercontent.com/49913958/196328672-828f4f35-c905-499a-a24e-7e7f85757c78.png">

**Comment by baptistecdr** [comment]
> When I click "download with aria2" on Safari, it shows "failed to add like". Same setting works will on Chrome. <img alt="Screenshot 2022-10-18 at 11 26 07" width="627" src="https://user-images.githubusercontent.com/49913958/196328659-fd129f04-035d-4e21-8525-14ad5ef3b2dc.png"> <img alt="Screenshot 2022-10-18 at 11 01 17" width="370" src="https://user-images.githubusercontent.com/49913958/196328672-828f4f35-c905-499a-a24e-7e7f85757c78.png">

Yes, same, I will update the extension later. Sorry for the inconvenience.

**Comment by myd986** [comment]
> > When I click "download with aria2" on Safari, it shows "failed to add like". Same setting works will on Chrome. <img alt="Screenshot 2022-10-18 at 11 26 07" width="627" src="https://user-images.githubusercontent.com/49913958/196328659-fd129f04-035d-4e21-8525-14ad5ef3b2dc.png"> <img alt="Screenshot 2022-10-18 at 11 01 17" width="370" src="https://user-images.githubusercontent.com/49913958/196328672-828f4f35-c905-499a-a24e-7e7f85757c78.png">
> 
> Yes, same, I will update the extension later. Sorry for the inconvenience.

Thank for your work. Waiting for your good news.
